modules/openapi/api/apis/openapi/component_protocol/render.go

```diff
@@ -32,6 +32,7 @@ import (
 	"github.com/erda-project/erda/pkg/httpclient"
 	"github.com/erda-project/erda/pkg/httpserver"
 	"github.com/erda-project/erda/pkg/httpserver/errorresp"
+	i18nPkg "github.com/erda-project/erda/pkg/i18n"
```

Comment:
Use all lowercase in pkg name.

Reply:
You should merge "github.com/erda-project/erda/modules/openapi/i18n".

Reply:
i think it should merge "github.com/erda-project/erda/modules/openapi/i18n" later,or build another task or bug ...